configdemo/.env

```
@@ -0,0 +1,3 @@
+BUILD_ID=444444
+BUILD_NAME=NAME_TEST
+BUILD_VERSION=1.8.0
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)